ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -311,38 +322,45 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar, cl
-                // @ts-ignore
-                loadAsPerformanceProfile
-                // @ts-ignore
-                performanceProfileID={profileForModal?.id}
-                // @ts-ignore
-                profileName={profileForModal?.name}
-                // @ts-ignore
-                meshName={profileForModal?.service_mesh}
-                // @ts-ignore
-                url={profileForModal?.endpoints?.[0]}
-                // @ts-ignore
-                qps={profileForModal?.qps}
-                // @ts-ignore
-                loadGenerator={profileForModal?.load_generators?.[0]}
-                // @ts-ignore
-                t={profileForModal?.duration}
-                // @ts-ignore
-                c={profileForModal?.concurrent_request}
-                // @ts-ignore
-                reqBody={profileForModal?.request_body}
-                // @ts-ignore
-                headers={profileForModal?.request_headers}
-                // @ts-ignore
-                cookies={profileForModal?.request_cookies}
-                // @ts-ignore
-                contentType={profileForModal?.content_type}
+              <Paper className={classes.addProfileModal} >
+                <div className={classes.dialogHeader}>
+          <DialogTitle className={classes.title} >Performance Profile Wizard</DialogTitle>
+                <IconButton aria-label="close" style={{color: "white"}} onClick={() => setProfileForModal(undefined) }>
+                  <CloseIcon />
+                </IconButton>
+                </div>
+
+                <MesheryPerformanceComponent
                 // @ts-ignore
-                runTestOnMount={!!profileForModal?.runTest}
-              />
-            </Paper>
+                  loadAsPerformanceProfile
+                  // @ts-ignore
+                  performanceProfileID={profileForModal?.id}
+                  // @ts-ignore
+                  profileName={profileForModal?.name}
+                  // @ts-ignore
+                  meshName={profileForModal?.service_mesh}
+                  // @ts-ignore
+                  url={profileForModal?.endpoints?.[0]}
+                  // @ts-ignore
+                  qps={profileForModal?.qps}
+                  // @ts-ignore
+                  loadGenerator={profileForModal?.load_generators?.[0]}
+                  // @ts-ignore
+                  t={profileForModal?.duration}
+                  // @ts-ignore
+                  c={profileForModal?.concurrent_request}
+                  // @ts-ignore
+                  reqBody={profileForModal?.request_body}
+                  // @ts-ignore
+                  headers={profileForModal?.request_headers}
+                  // @ts-ignore
+                  cookies={profileForModal?.request_cookies}
+                  // @ts-ignore
+                  contentType={profileForModal?.content_type}
+                  // @ts-ignore
+                  runTestOnMount={!!profileForModal?.runTest}
+                />
```

Comment:
Do we still need `// @ts-ignore` after `// @ts-nocheck` at the top of the file. Since `// @ts-nocheck` ignore the whole file.